src/internet_identity/tests/tests.rs

```diff
@@ -1896,18 +1896,20 @@ mod http_tests {
         let canister_id = install_ii_canister(&env, II_WASM.clone());
 
         let metrics = flows::get_metrics(&env, canister_id);
-        let (signature_count, _) = parse_metric(&metrics, "internet_identity_stable_memory_pages");
+        let (stable_memory_pages, _) =
+            parse_metric(&metrics, "internet_identity_stable_memory_pages");
         // empty II has some metadata in stable memory which requires at least one page
-        assert_eq!(signature_count, 1);
+        assert_eq!(stable_memory_pages, 1);
 
-        // a wasm page is 64kb and a single user takes up 2kb -> 32 users require a complete wasm page
-        for _ in 0..32 {
+        // a wasm page is 64kb and a single user takes up  2kb -> 33rd anchor record starts on the second wasm page
+        for _ in 0..33 {
```

Comment:
Can you explain why this had to be bumped? I'm guessing 32 anchors just fit in one page; why did the test pass before?

Reply:
The reason is, that by using stable structures `Writer`, we now only grow the memory on write, where beforehand we would always grow the memory to fit the whole 2048 bytes an anchor record might use (regardless of the actual size).

And because the anchor does not actually use the whole 2048 bytes, this caused it to still fit into the previous memory page. So we had to bump it in order to cause a write to page 2.